pages/privacy.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: page
+title: Privacy Policy
+---
+
+MDAnalysis is a [fiscally sponsored
+project]({{site.numfocus.sponsored_project}}) of [NumFOCUS][], a nonprofit
+dedicated to supporting the open source scientific computing
+community. Like NumFOCUS, we value the privacy of our users.
+
+
+## NumFOCUS Privacy Policy
+
+**All MDAnalysis websites are legally governed by the [NumFOCUS
+Privacy Policy][]**, which can be found at
+[https://numfocus.org/privacy-policy](https://numfocus.org/privacy-policy).
+
+The sections below explain how MDAnalysis processes and collects data,
+which differs from sites of the NumFOCUS organization.
+
+
+## What information does MDAnalysis collect?
+
+We only collect anonymous per-page *usage statistics* using
+[GoatCounter][] with details explained in the [GoatCounter privacy
+policy][]. These statistics are publicly visible at
+[https://mdanalysis.goatcounter.com](https://mdanalysis.goatcounter.com/).
+
+*Online payments* (donations) are processed through NumFOCUS; as soon
+as you follow a "donate" button or link on a MDAnalysis website, you
+will be transferred to a NumFOCUS site. Please see the [NumFOCUS
+Privacy Policy][] regarding data collected during payment processing.
+
+
+## Who will your information be shared with?
+
+We only share information with the following third parties:
+
+* Cloud computing services:
+  [Cloudflare](https://www.cloudflare.com). Requests are routed
+  through Cloudflare's caches and DNS resolvers; please see
+  [https://www.cloudflare.com/privacypolicy](https://www.cloudflare.com/privacypolicy)
+  for how Cloudflare collects information.
+* Search: [algolia](https://www.algolia.com/) provides search
+  functionality. Your search terms are transmitted to algolia to
+  provide the search functionality on our site. Please see
+  [https://www.algolia.com/policies/privacy](https://www.algolia.com/policies/privacy/)
+  (9. Subscriber Data submitted to our Services or collected through
```

Comment:
This looks like copy-paste leftover.

Reply:
"(9. Subscriber Data submitted to our Services or collected through" is the section in the algolia policy that governs the data – but I agree, that's not clear.

Reply:
I can remove it (clean solution, similar to what NF does, who only link to the 3rd party policies) or say "look under xxx in {link} to learn more". Any preference?

Reply:
I would just keep the link. Maybe somebody else has a different opinion.

Reply:
Starting from @lilyminium 's edit I formatted it as

> Search: [algolia](https://www.algolia.com/) provides search functionality. Your search terms are transmitted to algolia to   provide the search functionality on our site. Please see  [https://www.algolia.com/policies/privacy](https://www.algolia.com/policies/privacy/)  under Section *9. Subscriber Data submitted to our Services or collected through  our Services (on behalf of our subscribers)* to learn how algolia  uses your data.

Is this clearer or should I just delete it and keep the link? Feel free to add a suggested code change so that I can press a button.